index.html

```diff
@@ -0,0 +1,186 @@
+							<li class="nav-link">English</li>
+							<li class="nav-link">My Page</li>
+							<li class="nav-link">Logout</li>
+						</ul>
+					</div>
+				</div>
+				<ul class="nav-items isHidden">
+					<i class="close-icon material-icons">close</i>
+					<li class="nav-item"><a href="/pages/about.html">About</a></li>
+					<li class="nav-item"><a href="#main-program">Program</a></li>
+					<li class="nav-item"><a href="#">Join</a></li>
+					<li class="nav-item"><a href="#partners">Sponsor</a></li>
+					<li class="nav-item"><a href="#">News</a></li>
+					<li class="cc"><a>CC Campaign</a></li>
+				</ul>
+				<nav class="desktop-nav-items">
+					<a href="/" class="wd-logo">WD<span>Course</span></a>
+					<div class="menu-cc">
+						<ul class="desktop-nav-items-links">
+							<li class="desktop-nav-item-link about"><a href="/pages/about.html">About</a></li>
+							<li class="desktop-nav-item-link"><a href="#main-program">Program</a></li>
+							<li class="desktop-nav-item-link"><a href="#">Join</a></li>
+							<li class="desktop-nav-item-link"><a href="#partners">Sponsor</a></li>
+							<li class="desktop-nav-item-link"><a href="#">News</a></li>
+						</ul>
+						<a class="desktop-cc">CC Campaign</a>
+					</div>
+				</nav>
+			</nav>
+		</header>
+
+		<main>
+			<section id="hero" class="hero">
+				<div class="heading">
+					<p class="hello">"Hello! Learner world"</p>
+					<h2>Build websites and applications with the best</h2>
+				</div>
+				<div class="desc">
+					<p>The world of web development is as wide as the internet itself. Much of our social and vocational lives
+						play out on the internet, which prompts new industries aimed at creating, managing, and debugging the
+						websites and applications that we increasingly rely on.</p>
+				</div>
+				<div class="eventDate">
+					<p>2023.06.28(WED) ~ 30(FRI)</p>
+					<small>@ Jecode93 | Software Engineer and more</small>
+				</div>
+			</section>
+
+			<section id="main-program" class="main-program">
+				<div class="program-heading">
+					<h1>Main Program</h1>
+					<div class="trait"></div>
+				</div>
+				<div class="program-contents">
+					<!-- Program 1 -->
+					<div class="program-content">
+						<div class="program-icon">
+							<img src="./images/program-icon-01.png" alt="Conference icon">
+						</div>
+						<div class="program-h2">
+							<h2>Lecture</h2>
+						</div>
+						<div class="program-p">
+							<p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+						</div>
+					</div>
+					<!-- Program 2 -->
+					<div class="program-content">
+						<div class="program-icon">
+							<img src="./images/program-icon-02.png" alt="Conference icon">
+						</div>
+						<div class="program-h2">
+							<h2>CC Exhibition</h2>
+						</div>
+						<div class="program-p">
+							<p>Appreciate various creations applying CC license of artists, organized from at center Nabi</p>
+						</div>
+					</div>
```

Comment:
- [ ] Semantically speaking, I think it would be nice if you can use the `<article>` tag instead the `<div>` one as the card container. This is because each of these cards represents a self-contained composition and the `<article>` tag provides just this kind of information to the browser and screen readers, which improves the accessibility of your site! 🤓 🔧 

```suggestion
					<article class="program-content">
						<div class="program-icon">
							<img src="./images/program-icon-02.png" alt="Conference icon">
						</div>
						<div class="program-h2">
							<h2>CC Exhibition</h2>
						</div>
						<div class="program-p">
							<p>Appreciate various creations applying CC license of artists, organized from at center Nabi</p>
						</div>
					</article>
```